lib/jazzy/config.rb

```diff
@@ -400,6 +401,12 @@ def print_attr_description(attr)
       end
     end
 
+    class PathnameArray < Array
```

Comment:
I understand the code smell, but I don’t see the better way. What do you mean by “adding a path attr to the config options?”